nixos/modules/services/misc/matrix-synapse.nix

```diff
@@ -687,7 +691,7 @@ in {
         WorkingDirectory = cfg.dataDir;
         PermissionsStartOnly = true;
         ExecStart = ''
-          ${cfg.package}/bin/homeserver \
+          ${python.interpreter} -m synapse.app.homeserver \
```

Comment:
This should probably be squashed into the commit that removed the script.

Reply:
what commit do you mean ?